tests/torch/sparsity/movement/test_model_saving.py

```diff
@@ -268,9 +268,8 @@ def test_ngraph_pruning(self, tmp_path: Path, desc: dict):
         pruned_file_bytes = pruned_file.stat().st_size
         not_pruned_file_bytes = not_pruned_file.stat().st_size
         file_size_ratio = 1 - pruned_file_bytes / not_pruned_file_bytes
-        assert pytest.approx(compression_rate, abs=1e-2) == desc.nncf_weight_ratio
         assert (
-            pytest.approx(file_size_ratio, abs=1e-2) == desc.ov_weight_ratio
+            pytest.approx(file_size_ratio, abs=3e-2) == desc.ov_weight_ratio
```

Comment:
Is there any reason to increase the approximate value? Were the reasons for the changes analysed properly?

Reply:
<img width="646" alt="image" src="https://github.com/user-attachments/assets/1ccb2ff0-16fd-4521-a3af-0cb065d6736f">

Changes of size of file was a result of replaced `stridedslice` layer to `slice`, that collect extra argument `axis` as result size constant data for argument for this node was reduced.
It was done to improve performance of llm model in ticket: 144394